src/ResourceManager/Resources/Commands.Resources/Properties/Resources.resx

```diff
@@ -243,6 +243,27 @@
   <data name="ResourceProviderNotFound" xml:space="preserve">
     <value>Could not find the resource provider '{0}'.</value>
   </data>
+  <data name="ScopeShouldBeginWithSlash" xml:space="preserve">
+    <value>Scope '{0}' must begin with '/'.</value>
+  </data>
+  <data name="ScopeShouldBeginWithSubscriptionsAndResourceGroups" xml:space="preserve">
+    <value>Scope '{0}' should begin with '/subscriptions/&lt;subid&gt;/resourceGroups'.</value>
```

Comment:
IMO, it is better to point out the error step by step, user can easily learn which part is wrong.

---
In reply to: [119960272](https://github.com/Azure/azure-powershell/pull/4019#discussion_r119960272) [](ancestors = 119960272)